### PR TITLE
Kleingruppe

### PR DESCRIPTION
**Pull request by nzjony**
Hi @simonhaenisch ,

I managed to get the google map working and the AJAX email being sent.

To test, remove the "disabledgroups:" in berlin/kleingruppen.html

Because we have still a week or so, there is no rush to review, feel free to be thorough in your code review :)

Thanks a lot!

Jonathan

### CODE REVIEW

File: berlin/kleingruppen.html [hunk 1]
```diff
@@ -6,19 +6,348 @@
   html: <h1>Kleingruppen</h1>
   image: /uploads/egroups-hero.jpg
   image_position: center 25%
+emails: jonathan.stichbury@equippers.de,jonotkd@gmail.com
```
Comment (nzjony) [inline berlin/kleingruppen.html]: feel free to use this for testing, I will change in a separate commit

File: berlin/kleingruppen.html [hunk 2]
```diff
@@ -6,19 +6,350 @@
...
+    leader: 'Regine & Marc'
+    target: gemischt
+    description: 'Gemeinschaft, gemeinsames Abendessen, Bibelstudium zu einem Buch der Bibel'
+    date: 'dienstags, 19 Uhr'
+    location: Berlin-Gesundbrunnen
+    max: 10
+    full: false
+    lat: 52.5657462
+    lng: 13.3910762
+    img: sina.png
```
Comment (simonhaenisch) [inline berlin/kleingruppen.html]: really? or is it just a bad file name?

File: berlin/kleingruppen.html [hunk 3]
```diff
@@ -6,19 +6,350 @@
...
+
+        var request = new XMLHttpRequest();
+
+        request.addEventListener("load", function () {
+          if (request.status === 302 || // CloudCannon redirects on success
+              request.status == 200 ) { // Or testing showed that it didn't
+            alert("Nachricht gesendet.");
+          }
+          else {
+            alert("Fehler beim Senden der Nachricht. Bitte versuchen Sie es später erneut");
```
[the remark this answers is not in the document]
Comment (nzjony) [reply berlin/kleingruppen.html]: I wrote it via google translate, danke.

File: berlin/kleingruppen.html [hunk 4]
```diff
@@ -6,19 +6,350 @@
...
+    <p>Im Moment sind wir mitten in einer Kleingruppen-Saison. Wenn du den Startschuss verpasst hast oder aus irgendwelchen Gründen gerade nicht dabei sein kannst, kein Problem. Mit Beginn der nächsten Saison bietet sich dir eine neue Chance. Also merk dir schon mal vor:</p>
+    <h4 class="uppercase">Nächste Kleingruppen-Saison: 17. September bis 9. Dezember.</h4>
+    <p>Hast du Fragen zu unseren Kleingruppen, dann wende dich an uns: kleingruppe@equippers.de. Wir kommen gern mit dir in Kontakt. Wenn du in der nächsten Saison eine eigene Kleingruppe anbieten möchtest, sprich uns gern sonntags nach dem Gottesdienst persönlich oder am Info-Punkt an.</p>
+  </section>
 
-		<p>Im Moment sind wir mitten in einer Kleingruppen-Saison. Wenn du den Startschuss verpasst hast oder aus irgendwelchen Gründen gerade nicht dabei sein kannst, kein Problem. Mit Beginn der nächsten Saison bietet sich dir eine neue Chance. Also merk dir schon mal vor:</p>
+  <section>
+    {% if page.smallgroups.size > 0 %}
```
[the remark this answers is not in the document]
Comment (nzjony) [reply berlin/kleingruppen.html]: chur

File: berlin/kleingruppen.html (hunk 2 above)
Comment (nzjony) [reply berlin/kleingruppen.html]: They are just test data, I will remove and put in the proper data shortly.

**Comment by nzjony** [comment]
hi @simonhaenisch, can you check my latest changes?

**Comment by simonhaenisch** [comment]
want me to merge already? or when is it supposed to go live?

**Comment by simonhaenisch** [comment]
Nah, you can just wait till the end of the week then, and click the "Merge pull request" button yourself 😉

**Pull request merged**